### ISSUE TITLE
train model on the casia-50w dataset,the model accuracy is very low

### ISSUE THREAD

**Issue by onnx20** [DanJun6737/TopoFR#1]
Firstly, thank you very much for your contributions！

the configuration file is modified based on glint360k_r50.py
`from easydict import EasyDict as edict

config = edict()
config.margin_list = (1.0, 0.5, 0.0)
config.network = "r50"
config.resume = False
config.output = 'casia-webface-50w-r50'
config.embedding_size = 512
config.sample_rate = 1.0
config.fp16 = True
config.momentum = 0.9
config.weight_decay = 1e-4
config.batch_size = 256
config.lr = 0.1
config.verbose = 2000
config.dali = False

config.rec = "/Data/face_data/casia-webface-50w_rec"
config.num_classes = 10572
config.num_image = 494149
config.num_epoch = 25
config.warmup_epoch = 0
config.val_targets = ['lfw', 'cfp_fp', "agedb_30"]
config.num_workers = 4`

train logs:
![train](https://github.com/user-attachments/assets/311143f4-fb68-4169-8d7f-172dba749ac7)

MFR test result:
![MFR](https://github.com/user-attachments/assets/f1aead3f-2bb9-45c4-9464-be4b4acc287f)

What may be the possible reasons for the low accuracy of the model？ Thanks.

**Comment by DanJun6737** [comment]
Hi~ @onnx20 
The configuration file we offer is designed for training models with 4 GPUs. If you opt for a different number of GPUs, I recommend adjusting certain parameters like batch size, learning rate, and others accordingly. Inappropriate parameters can also hinder the model from converging effectively.
Moreover, you can print and observe the topology structure alignment loss value. If it is significant, consider appropriately reducing its coefficient.

**Comment by onnx20** [comment]
Thanks for your response!@DanJun6737 
I use the configuration file of 4 GPUs(glint360k_r50.py), and only the dataset is different(casia-webface). Finally, the same low precision result was obtained. May I ask which parameters I can modify?
![train](https://github.com/user-attachments/assets/49ab9e1b-8eba-4e3b-9d0f-a26978e5c5f2)

I also adjusted the learning rate and batch parameters, but the loss still cannot converge. Thanks.

**Comment by onnx20** [comment]
@DanJun6737 
![loss](https://github.com/user-attachments/assets/accf65f4-1357-490e-a705-f458112b4eac)
the loss_topo converges normally , but the loss_cls still cannot converge.

**Comment by DanJun6737** [comment]
@onnx20 Hi
I suggest referring to the configuration files in [InsightFace](https://github.com/deepinsight/insightface/tree/master/recognition/arcface_torch/configs) to adjust the parameters of your config file.
The failure of the loss function " loss_vls" to converge is attributable to the configuration parameters.
